docsite/rst/playbooks_lookups.rst

```diff
@@ -368,6 +368,29 @@ TLSA         usage, selector, mtype, cert
 TXT          strings
 ==========   =============================================================================
 
+.. _credstash_lookup:
+
+The etcd Lookup
+````````````````````
+.. versionadded:: 2.3
+
+When you need to query a value from an etcd server you can use etcd lookup. Until version 2.3 you could interrogate localmachine by default or using environment variable you could interrogate a remote v1 server. Since 2.3 this functionality has been improved by allowing to set the etcd host and version directly from you playbook:
+
+Example usage:
+
+# query localhost etcd ( v1 )
```

Comment:
If you indent this sample block, like in "Here are some examples" under "More Lookups" that should fix the formatting